### PR TITLE
make connection configurable from yaml

### PR DESCRIPTION
**Pull request by farmio**
Auto, tunneling or routing and their associated ip-addresses can be configured via xknx.yaml
#130

### CODE REVIEW

File: xknx.yaml [hunk 1]
```diff
@@ -1,8 +1,11 @@
 
 general:
     own_address: '15.15.249'
-    rate_limit: 20
+    rate_limit: 18
```
Comment (marvin-w) [inline xknx.yaml]: Why did you change this? Testing?

Comment (farmio) [reply xknx.yaml]: right. just for the tests to have a different value than default. 🤔maybe we should rename this file to something like `test.yaml` or `example.yaml`? Or just write some comments to point that out.

Comment (marvin-w) [reply xknx.yaml]: Yea maybe, From what I know it was just an example so maybe example.yml might do the trick. Anyway, as it's unrelated to the contents of this PR i will resolve this conversation
